emukit/core/optimization/anchor_points_generator.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2016, the GPyOpt Authors
```

Comment:
I am not sure if this should remain like that, or be changed to emukit header. Let's research before we push this PR

Reply:
Good point, I completely missed this header and didn't think about whether it should stay

Reply:
As the current maintainer of GPyOpt, I grant emukit permission to claim copyright to this file. Feel free to change to header to emukit's